src/buildDictionary.ts

```diff
@@ -1,3 +1,4 @@
+import { StripXMLTagsFromTranslation } from './translationPipes/StripXMLTagsFromTranslation.js'
```

Comment:
I think import of our sources should be after packages.

Reply:
Sure, fixed, later we can automate this with a prettier plugin: https://github.com/trivago/prettier-plugin-sort-imports